dev/examples/controls/line.js

```diff
@@ -523,3 +523,11 @@ export const renderDashedLine = (id) => {
     lineDefault.loadContent(Carbon.api.line(axisData.data[1]));
     return lineDefault;
 };
+export const renderLegendWithOptions = (id) => {
```

Comment:
`renderLineWithLegendOptions` 
Lets make this consistent with others.

Reply:
Done